src/Behat/Testwork/Exception/Stringer/PHPUnitExceptionStringer.php

```diff
@@ -35,7 +35,11 @@ public function supportsException(Exception $exception)
      */
     public function stringException(Exception $exception, $verbosity)
     {
-        if (!class_exists('PHPUnit\\Framework\\TestFailure')) {
+        if (class_exists('PHPUnit\\Util\\ThrowableToStringMapper')) {
+            return trim(\PHPUnit\Util\ThrowableToStringMapper::map($exception));
+        }
+
+        if (class_exists('PHPUnit_Framework_TestFailure')) {
```

Comment:
this changed the behavior when both `PHPUnit_Framework_TestFailure` and `PHPUnit\Framework\TestFailure` exist.